config/fxdata/terrain.cfg

```diff
@@ -1086,11 +1086,22 @@ Messages = 0 0 0
 ; Sound sample playing in a loop when hovering over it on screen.
 AmbientSndSample = 0
 ; Algorithm used to compute room capacity:
-; slabs_all_only, slabs_all_wth_effcncy, slabs_div2_wth_effcncy, gold_slabs_wth_effcncy, gold_slabs_full, none.
+; slabs_all_only            - One capacity per slab
+; slabs_all_wth_effcncy     - Capacity is reduced when the room is less effecient
+; slabs_no_min_wth_effcncy  - 
+; slabs_div2_wth_effcncy    - Capacity is reduced by room effeciency, then divided by two.
+; slabs_div2_nomin_effcncy  - 
+; slabs_mul2_wth_effcncy    - 
+; slabs_pow2_wth_effcncy    - 
+; gold_slabs_wth_effcncy    - Capacity is reduced by room effeciency, then multiplied by gold hoard types (5)
+; gold_slabs_full           - One capacity per slab, then multiplied by gold hoard types
+; none                      - No capacity
 TotalCapacity = none
 ; Algorithm used to used storage space and used worker space:
 ; count_gold_hoardes_in_room, count_books_in_room, count_workers_in_room, count_crates_in_room, count_bodies_in_room, count_food_in_room, count_lair_occupants, none.
 UsedCapacity = none none
+; Slab that it also receives full efficiency bonus from.
+SlabSynergy = none
```

Comment:
Probably because the functions have a 'none' function, but indeed since SlabSynergy takes a room, 'Null' would be most appropriate.

Reply:
Also because there is a NULL room, but not a NULL slab. If I set to NULL, it will still give back log errors, I tried that. But maybe I did something wrong? I'm still very new to this codebase.